compiler/rustc_public/src/lib.rs

```diff
@@ -56,6 +57,7 @@ pub mod visitor;
 pub type Symbol = String;
 
 /// The number that identifies a crate.
+// FIXME: Make this a newtype, so it can have a `ReferencesTls`
```

Comment:
why not include in this PR?

Reply:
I wanted to keep this PR not doing too much at once. But I've added it in 8b3ee9f